### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by p-rdx**

### CODE REVIEW

File: deeppavlov_agent/core/agent.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                 pipeline: Pipeline,
+                 state_manager: StateManager,
+                 workflow_manager: WorkflowManager,
+                 response_logger: BaseResponseLogger) -> None:
+        self.pipeline = pipeline
+        self.state_manager = state_manager
+        self.workflow_manager = workflow_manager
+        self._response_logger = response_logger
+
+    def flush_record(self, dialog_id: str):
```
Comment (yoptar) [inline deeppavlov_agent/core/agent.py]: return types for methods?

File: deeppavlov_agent/run.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import argparse
+import json
+import logging
+
+import yaml
+from aiohttp import web
+import os
```
Comment (yoptar) [inline deeppavlov_agent/run.py]: imports order

File: deeppavlov_agent/state_formatters/__init__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from .dp_formatters import *
+from .output_formatters import *
```
Comment (yoptar) [inline deeppavlov_agent/state_formatters/__init__.py]: Are you sure?

File: deeppavlov_agent/state_formatters/dp_formatters.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        model_args_names[3]: [dialog]
+    }
+
+
+def ner_formatter_out(payload: List):
+    if len(payload) == 2:
+        return {'tokens': payload[0],
+                'tags': payload[1]}
+    else:
+        raise ValueError("Payload doesn't contain all required fields")
```
Comment (yoptar) [inline deeppavlov_agent/state_formatters/dp_formatters.py]: What if there are more fields than it's required?
Maybe just do something like:
```python3
def ner_formatter_out(payload: List):
    tokens, tags = payload
    return {'tokens': tokens, 'tags': tags}
```

File: deeppavlov_agent/core/agent.py (hunk 1 above)
Comment (p-rdx) [reply deeppavlov_agent/core/agent.py]: Later

**Pull request merged**